### PR TITLE
Bl/#1294 implement anchored time mutator 4ci

### PR DESCRIPTION
**Pull request by bhavyalatha26**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/1325)

### CODE REVIEW

File: src/main/resources/beam-template.conf [hunk 1]
```diff
@@ -150,6 +150,7 @@ beam.replanning.ModuleProbability_3=0.1
 beam.replanning.Module_4="TimeMutator"
 beam.replanning.ModuleProbability_4=0.0
 beam.replanning.fractionOfIterationsToDisableInnovation = "double | Double.PositiveInfinity"
+beam.replanning.anchorEndTimeMutator.mutation.range=1
```
Comment (wrashid) [inline src/main/resources/beam-template.conf]: rangeInSeconds - default: 1800

File: src/main/scala/beam/replanning/AnchorEndTimeMutator.scala [hunk 2]
```diff
@@ -0,0 +1,73 @@
+package beam.replanning
+
+import beam.sim.config.BeamConfig
```
Comment (wrashid) [inline src/main/scala/beam/replanning/AnchorEndTimeMutator.scala]: we need to enable usage of this replanning module as well

File: src/main/resources/beam-template.conf (hunk 1 above)
Comment (bhavyalatha26) [reply src/main/resources/beam-template.conf]: done

File: src/main/scala/beam/replanning/AnchorEndTimeMutator.scala (hunk 2 above)
Comment (bhavyalatha26) [reply src/main/scala/beam/replanning/AnchorEndTimeMutator.scala]: done

**Comment by bhavyalatha26** [comment]
test!

**Pull request closed without merging**